variables.tf

```diff
@@ -233,7 +198,7 @@ variable "mount_points" {
   default     = []
 }
 
-variable "environment" {
+variable "list_environment" {
```

Comment:
I think this should be called container_environment and the one below called map_container_environment

Reply:
@joe-niland fixed